assignment/src/com/assignmentweek2/person/Person.java

```diff
@@ -0,0 +1,13 @@
+package com.assignmentweek2.person;
+/*
+This is the abstract class
+ */
+public abstract class Person {
+        abstract void Studentdisplay();
+        abstract void Teacherdisplay();
+        abstract void Doctordisplay();
+        abstract void StudentAddress();
+        abstract void TeacherAddress();
+        abstract void DoctorAddress();
```

Comment:
As Person class is an abstract class for all person types, method names should not be specific to any particular type. For example - use "display" instead of "studentDisplay".

Reply:
changed